.github/workflows/validate-this-gha-refs.yml

```diff
@@ -3,22 +3,97 @@ on:
   push:
     branches:
       - dev
+  pull_request:
   pull_request_review:
     types: [submitted]
-
+  
 jobs:
+  check-pr-reviews:
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-22.04
+    permissions:
+      actions: read
+      contents: read
+      pull-requests: read
+    outputs:
+      last-review-approved: ${{ steps.check-pr-reviews.outputs.last-review-approved }}
+    steps:
+      - name: Check PR Reviews
+        id: check-pr-reviews
+        uses: Lombiq/GitHub-Actions/.github/actions/check-pull-request-reviews@dev
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
   validate-gha-refs:
+    needs: check-pr-reviews
+    if: always()
```

Comment:
Good question. The `check-pr-reviews` job was conditional to only run on `pull_request` events so that job would be skipped on `push` or `pull_request_review` events, and consequently, the `validate-gha-refs` job would also be skipped even thought there were steps that needed to be run for the non-pull_request scenario.

I realized this was confusing. So I refactored the code to pull the `check-pr-review` job's [one and only step](https://github.com/Lombiq/GitHub-Actions/blob/1f1e6e2fd79fb06ae97963a039a7ddbfc3dfbe19/.github/workflows/validate-this-gha-refs.yml#L21-L25) into the `validate-gha-refs` job itself [here](https://github.com/Lombiq/GitHub-Actions/blob/issue/OSOE-517/.github/workflows/validate-this-gha-refs.yml#L62-L67).

And all the steps are now conditional based on the event that triggered the workflow run.